libs/csm/rest/csm_rest_quota.py

```diff
@@ -64,14 +64,16 @@ def get_user_quota(self, uid, **kwargs):
 
     #pylint disable=no-self-use
     def iam_user_quota_payload(self,enabled: str,
-                        max_size: int, max_objects: int):
+                        max_size: int, max_objects: int,
+                        check_on_raw: str):
```

Comment:
Can u add details of the parameter in the function comment. Also expected units for max_size?

Reply:
sure